Source/Modules/lang.cxx

```diff
@@ -378,12 +380,8 @@ cplus_runtime(0) {
      Default director 'protected' constructor code, disabled by
      default. Each language that needs it, has to define it.
    */
-  director_prot_ctor_code = 0;
```

Comment:
The comment just above is about this member variable and is just confusing with this line of code removed.  Probably best relocated to where the member is defined in `swigmod.h`.

Reply:
Done! @ojwb